meshtastic/tests/test_node.py

```
@@ -1496,21 +2644,28 @@ def test_deleteChannel_missing_or_out_of_range_validations(
     iface = autospec_local_node_iface(MeshInterface)
     anode = Node(iface, "!12345678", noProto=True)
     iface.localNode = anode
     anode.channels = [Channel(index=0, role=Channel.Role.SECONDARY)]
-    anode._channels_lock = _DropChannelsOnEnterCountLock(  # type: ignore[assignment]
-        anode, trigger_enter=2
-    )
     anode.ensureSessionKey = MagicMock()  # type: ignore[method-assign]
-    anode._send_admin = MagicMock()  # type: ignore[method-assign]
+    dropped_channels = False
+
+    def _drop_channels_on_first_send(*args: Any, **kwargs: Any) -> mesh_pb2.MeshPacket:
```

Ruff (0.15.5)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)